packaging/docker/build-images.sh

```diff
@@ -250,7 +250,7 @@ build_output_directory="${script_dir}/../../"
 source_code_diretory=$(awk -F= '/foundationdb_SOURCE_DIR:STATIC/{print $2}' "${build_output_directory}/CMakeCache.txt")
 commit_sha=$(cd "${source_code_diretory}" && git rev-parse --verify HEAD --short=10)
 fdb_version=$(cat "${build_output_directory}/version.txt")
-fdb_library_versions=( '6.3.25' '7.1.57' '7.3.37' "${fdb_version}" )
+fdb_library_versions=( '7.1.57' '7.3.37' "${fdb_version}" )
```

Comment:
Is there a reason we package the old versions into the same docker image? If so we might want to add that information here.

Reply:
Use 7.3.63 to be consistent with the other file change?

I guess it might be related to support upgrade? Not entirely sure about it.